codebasin/report.py

```diff
@@ -780,9 +780,12 @@ def files(
     for f in codebase:
         setmap = defaultdict(int)
         if state:
-            for node, assoc in state.get_map(f).items():
-                if isinstance(node, CodeNode):
-                    setmap[frozenset(assoc)] += node.num_lines
+            association = state.get_map(f)
+            for node in [
+                n for n in state.get_tree(f).walk() if isinstance(n, CodeNode)
```

Comment:
IMO this is pretty hard to read - I'd suggest breaking it up, or something like

```python
for node in filter(lambda x: isinstance(x, CodeNode), state.get_tree(f).walk()):
    ...
```

I don't know if this necessarily lowers the complexity by much, but I think at least the intention is clearer by using `filter`.

Reply:
Fixed in b0526d2f0cf32ec5df34b90f1df235d4900caa3e.